charts/github-bot-safe-settings/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: safe-settings
+description: A GitHub App built with [Probot](https://github.com/probot/probot) to manage policy-as-code and apply repository settings to repositories across an organization.
+type: application
+version: 0.1.0
+appVersion: "2.0.0"
+icon: https://wile.example.com/anvil.svg
+keywords:
+  - github
+  - policy
+home: https://github.com/github/safe-settings
+sources:
+  - https://github.com/github/safe-settings
+maintainers:
+  - name: Yadhav Jayaraman
+    email: decyjphr@github.com
```

Comment:
- why did we fork this chart instead of using what is already available?
- if we made changes - then we should become authors.

every chart should have these README.* files and the documentation should be generated in pre-commit phase. Pls install it when you work with this repo via `pre-commit install`
![image](https://github.com/saritasa-nest/saritasa-devops-helm-charts/assets/70822710/9f2f63fd-d3b7-45b5-95de-067898de7915)